fluvio/__init__.py

```diff
@@ -5,6 +5,32 @@
 This module builds on top of the Fluvio Client Rust Crate and provides a
 pythonic access to the API.
 
+
+Creating a topic with default settings (1 partition, 1 replication) is as simple as:
+
+```python
+fluvio_admin = FluvioAdmin.connect()
+fluvio_admin.create_topic("a_topic")
+```
+
+Or just create a topic with custom settings:
+
+```python
+import fluvio
+
+fluvio_admin = FluvioAdmin.connect()
+topic_spec = (
+    TopicSpec.create()
+    .with_partitions(3)
```

Comment:
Let's keep the examples shorter and just configure retention time and segment size in this doc.

Reply:
like this?


Creating a topic with default settings is as simple as:

```python
fluvio_admin = FluvioAdmin.connect()
fluvio_admin.create_topic("a_topic")
```

Or just create a topic with custom settings:

```python
import fluvio

fluvio_admin = FluvioAdmin.connect()
topic_spec = (
    TopicSpec.create()
    .with_retention_time("1h")
    .with_segment_size("10M")
    .build()
)
fluvio_admin.create_topic("a_topic", topic_spec)
```